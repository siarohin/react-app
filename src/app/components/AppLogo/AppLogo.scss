.app-logo {
  font-family: var(--app-font-family-light);
  color: var(--app-theme-data-viz-red);
  font-size: 2.4rem;
  text-transform: lowercase;
  margin: 0;
}
