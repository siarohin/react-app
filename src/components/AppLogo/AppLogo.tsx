import React, { ReactElement } from "react";

import "./AppLogo.scss";

export const AppLogo = ({ content }: { content: string }): ReactElement => <h1 className="app-logo">{content}</h1>;
