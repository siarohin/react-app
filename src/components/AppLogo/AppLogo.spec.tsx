import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";

import { AppLogo } from "./AppLogo";

describe("Components.AppLogo.AppLogo: ", () => {
  const title: string = "Hello Words";

  it("should render component", () => {
    const component = renderer.create(<AppLogo content={title} />).toJSON();
    expect(component).toMatchSnapshot();
  });

  it("should display title", () => {
    render(<AppLogo content={title} />);
    expect(screen.getByRole("heading")).toHaveTextContent(title);
  });
});
