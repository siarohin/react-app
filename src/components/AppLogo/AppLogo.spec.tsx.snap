// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components.AppLogo.AppLogo:  should render component 1`] = `
<h1
  className="app-logo"
>
  Hello Words
</h1>
`;
